### ISSUE TITLE
macbook pro 15" : Display "main" cannot be set to 1440x900

### ISSUE THREAD

**Issue by thoolb** [univ-of-utah-marriott-library-apple/display_manager#9]
Display_Manager v.1.0.1. / macOS 10.12.6 / macbook pro 8,2

After restore the macbook from a Timemachine Backup the screen resolution 1440 x 900 only was set and available with no external monitor connected. As soon the external screen got plugged-in the main macbook LCD resolution jumped to the not-scaled 1440 x 852 – though this screen was successfully set to 1440 x 900 before the system restore.

When I now try to set resolution with plugged in external monitor Display Manager gives this result:

```
xxx:~ xxx$ display_manager.py res 1440 900 main
/usr/local/bin/display_manager.py:281: DeprecationWarning: BaseException.message has been deprecated as of Python 2.6
  raise CommandExecutionError(e.message, command=self)
Error: Display "main" cannot be set to 1440x900
```

(whereas i was able to set the wanted 1440 x 900 with the tool SwitchRes (v.4.8.1)

**Comment by uurazzle** [comment]
@thoolb Thanks for the feedback. You mentioned restoring via a Time Machine backup on a specific system. Did you try this on another system and got similar results? Did the 1440 x 900 resolution work with Display Manger previously to the Time Machine restore on this system?

**Comment by thoolb** [comment]
@uurazzle 
No, I haven't tried Display Manager within another hardware or OS. 

Yes, it worked with 1440 x 900 for years without complaints (this mac is from late 2011). I am quite shure I haven't done it with Display Manager that time (and not in OS Sierra, too). 

Unfortunately I can't remember with which tool I initially made this resolution work, I just do remember it was not simple, I had to try several tools ("cssreen", "Resolution Menu", "SwitchRes"). Nowadays I still can set this scaled 1440x 900 with "Resolution Manager" but it results in a color-flickering of particular elements on screen (like negativ/positiv) when the external monitor is plugged-in, PRAM reset doesn't solve it. So I used SwitchRes recently after Display Manager did not work.
